db-scheduler-boot-starter/src/main/java/com/github/kagkarlsson/scheduler/boot/config/DbSchedulerProperties.java

```diff
@@ -98,6 +100,16 @@
     @DurationUnit(SECONDS)
     private Duration shutdownMaxWait = SchedulerBuilder.SHUTDOWN_MAX_WAIT;
 
+    /**
+     * <p>Which log level to use when logging task failures. Defaults to {@link LogLevel#DEBUG}.</p>
+     */
+    private LogLevel failureLogLevel = SchedulerBuilder.DEFAULT_FAILURE_LOG_LEVEL;
+
+    /**
+     * <p>Whether or not to log the {@link Throwable} that caused a task to fail.</p>
+     */
+    private boolean logStackTraceOnFailure = SchedulerBuilder.LOG_STACK_TRACE_ON_FAILURE;
+
```

Comment:
I was thinking if maybe these two properties should have the same prefix, so you can see they belong together in the spring properties file. Like `failureLoggerLevel`, `failureLoggerLogStacktrace` ?

Reply:
I was thinking the same but couldn't come up with a good name for the latter property 🙃  The ones that you proposed sound good though, code updated.